### ISSUE TITLE
Recognize values if cursor is still active in field

### ISSUE THREAD

**Issue by do4gr** [prisma/studio#1059]
### Bug description

If I click add record while the cursor is still active in a field with a newly entered value, that value is not submitted. That is unexpected behaviour to me. If the value is required, it will at least error and I'll be able to notice, but for optional values it will just silently drop the input and store the value as null. 

<img width="1416" alt="Screen Shot 2022-11-23 at 10 12 58" src="https://user-images.githubusercontent.com/16278407/203509074-837aafaa-bb9c-42ac-a873-3b3012d706ef.png">

### Environment & setup

- Link to Data Browser instance: 
- OS: 
- Browser: 
- Database:

**Comment by devklepacki** [comment]
True. This always happen to me and then I wonder why data that I've just entered is not there.

**Comment by igalklebanov** [comment]
Hey :wave:

Prisma ORM 7 introduces a brand new standalone [Studio](https://www.prisma.io/docs/orm/tools/prisma-studio), built from the ground up to work everywhere, with or without Prisma ORM. Consequently, all "old Studio" issues are being closed as there won't be any further work on it. :mega:

The new Studio is also available in the Prisma VS Code extension and in the Prisma Console application. :electric_plug:

Help shape, improve and fix the new Studio with us in this repository by submitting bug reports and feature requests. :construction_worker:
